main.py

```diff
@@ -2,61 +2,77 @@
 from services import Services
 from application import App
 from PyQt5.QtWidgets import QApplication
-
+import logging
+from datetime import date, timedelta
+import screen_brightness_control as sbc
+import os
+import stat
 
 BASE_URL = 'https://api.olhar.media/'
```

Comment:
You do not put the same constant in multiple places. Also, why should it ever be used outside of the `services`?

Reply:
✅Removed the bas url everywhere except `services.py`